### PR TITLE
Log js outputs

### PR DESCRIPTION
**Pull request by hbcarlos**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Looks great! Merging.

File: share/jupyter/voila/templates/base/log.macro.html.j2 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    console.error = (...args) => {
+        window.top.postMessage({ level: "error", msg: ["[VOILA]:", ...args] });
+        _error(...args);
+    };
+    
+    //console.log("test");
+    //console.debug("debug");
+    //console.info("info");
+    //console.warn("warn");
+    //console.error("error");
```
Comment (maartenbreddels) [inline share/jupyter/voila/templates/base/log.macro.html.j2:33]: I guess some leftover debug code?

Comment (hbcarlos) [reply share/jupyter/voila/templates/base/log.macro.html.j2:33]: Yes, you can remove it.

File: share/jupyter/voila/templates/base/log.macro.html.j2 [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const _info = console.info;
+    const _warn = console.warn;
+    const _error = console.error;
+
+    console.debug = (...args) => {
+        window.top.postMessage({ level: "debug", msg: ["[VOILA]:", ...args] });
+        _debug(...args);
+    };
+
+    console.info = console.info = (...args) => {
```
Comment (jtpio) [inline share/jupyter/voila/templates/base/log.macro.html.j2:14]: Similar to the leftover comments mentioned in https://github.com/voila-dashboards/voila/pull/671/files#r471981963, we could also remove the superfluous second assignment to `console.info` in a follow-up PR.

**Pull request merged**